spec/controllers/articles_controller_spec.rb

```diff
@@ -1,47 +1,75 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
-    describe 'show the list to articles saved' do
+  describe 'GET #index' do
+    it 'renders the index template' do
+      get :index
+      expect(response).to render_template(:index)
     end
   end
 
-  describe '#show' do
-    it 'show the articles' do
+  describe 'GET #show' do
+    let(:article) { FactoryGirl.create(:article) }
+
+    it 'assigns the article with a given id' do
+      get :show, id: article.id
+      expect(assigns(:article)).to eq(article)
+    end
+
+    it 'renders the article view' do
+      get :show, id: article.id
+      expect(response).to render_template(:show)
+    end
+  end
+
+  describe 'GET #new'do
+    let(:article) { FactoryGirl.build(:article) }
+
+    it 'assigns an article' do
+      get :new
+      expect(assigns(:article)).to be_a_new(Article)
+    end
+  end
+
+  describe 'POST #create' do
+    let(:article) { FactoryGirl.create(:article)}
+
+    it 'renders the new artice' do
+      post :create, title: article.title
```

Comment:
I think than this is not necessary. Because you are using `let` with `FactoryGirl.create`.
This call factory to create automatically an article object when it is needed.